### ISSUE TITLE
research.branding(names)

### ISSUE THREAD

**Issue by skrpt** [feynman-project/blue-steel#1]
- [ ] List 50 total possible names for branding and company

**Comment by PhilipWilburn** [comment]
Test

**Comment by PhilipWilburn** [comment]
These are in no particular order:
1.The FIT
2.FIT 
3.PERFECT FIT
4.LLYL (LOOK LIKE YOU LIFT)
5. The Source
6. Appropriate Stitch
7. He Lifts Clothing
8. Big Bi's and Tri's
9. Jacked Fit
10. Ready Wear
11. Fit Button
12. Built to Suit
13. Built Like You
14. Swole Threads
15. SwoMoFoes
16. BLZ (Built like Zeus)
17. Titan
18. Get Fitted
19. Muscle Button
20. ToFit
21. Colossal 
22.  He Who Lifts
23. PLIFS (PHIL'S LIKE IT FITS SHIRTS) 
24.
25.

**Comment by PhilipWilburn** [comment]
I am really trying to determine what to go with because if our demo is professional businessmen who lift then the name needs to be a little more elegant. If our target is bodybuilding meatheads then something crazy like beast mode apparel would work.

**Comment by hotpok3t** [comment]
Just my opinion, I think your target market should be professionals that lift, not just meatheads. It's a great idea. Perhaps a mix of elegant, yet beast.

**Comment by PhilipWilburn** [comment]
I like that Chris. Finding a way to mix both is a really good idea. Thanks for the help.

**Comment by skrpt** [comment]
I agree with Chris' notion of catering to a wider array of professionals that can encompass both men and women alike.

**Comment by hotpok3t** [comment]
I recommend reading the original version of _The Art of The Start_ by Guy Kawasaki. It's part of the required reading for this entrepreneurship idea generation course I'm taking. The following course will be idea validation. Anyways, the book starts off by describing how ineffective most mission statements are. They are long, overly specific, and hard to remember. He recommends to instead first create a mantra. Nike's internal mantra for their employees is "Authentic athletic performance".  Their mission statement is "To bring inspiration and innovation to every athlete* in the world." Worry about creating the mission statement later. Describe what your goal is with this company in a 3-5 word mantra. 
 
Also you may consider just using an acronym for the name like YSL, PacSun, H&M, etc. Elegant doesn't need to be in the name, it should show in the quality of fit and materials. 

The wife and I are starting a clothing company called Blue Thistle after one of the flowers we had in our wedding.
